sensors/cmd/Dockerfile

```diff
@@ -1,20 +1,7 @@
-FROM centos:7
-RUN yum -y install ca-certificates openssh openssh-server openssh-clients openssl-libs curl git
-
-# OpenFass CLI
-COPY assets/faas-cli /usr/local/bin/faas
-
-# Argo Workflow CLI
-COPY assets/argo-linux-amd64 /usr/local/bin/argo
```

Comment:
we need argo cli. you can get rid of OpenFaas cli

Reply:
got it, thanks!

Reply:
done.